src/shaders/_prelude_fog.vertex.glsl

```diff
@@ -32,7 +32,7 @@ vec3 fog_position(vec2 pos) {
     return fog_position(vec3(pos, 0));
 }
 
-void fog_haze(vec3 pos, out float fog_opac, out vec4 haze) {
+void fog_haze(vec3 pos, out float fog_opac, out vec3 haze) {
```

Comment:
Just noticing this now, and not a change from this PR, but did we not name this function appropriately? As the output is both opacity and haze naming it fog_haze seems counterintuitive, it might be an overlook from the optimization when moving things fragment -> vertex.